doc/releasenotes/17_0_0_summary.md

```diff
@@ -160,3 +161,7 @@ Some notes to help understand these metrics:
  * `DurationByPhaseSeconds["CatchUpReplication"]` measures how long it took to catch-up replication after the restore phase.
  * `DurationByPhaseSeconds["RestoreLastBackup"]` measures to the duration of the restore phase.
  * `RestoreDurationNanoseconds["-.-.Restore"]` also measures to the duration of the restore phase.
+
+## <a id="deprecations-and-deletions"/> Deprecations and Deletions
```

Comment:
fully prepared to accept the consequence i'm going to conflict with my own PR (#12427) here

Reply:
The merge conflict above is on purpose?

Reply:
oh weird, i must have saved before VSCode finished actually doing the resolution. no this wasn't on purpose